src/common/styles/componentsStyles.ts

```diff
@@ -70,12 +79,21 @@ export const Input = styled.input`
   width: 14rem;
   height: 2rem;
   padding: 0.5rem;
+
+  @media (max-width: 540px) {
```

Comment:
it's better when all the media queries are consistent and coming from your theme